packit/config/aliases.py

```diff
@@ -119,6 +131,9 @@ def get_build_targets(*name: str, default: str = DEFAULT_VERSION) -> Set[str]:
                 for sys_and_version in get_versions(f"{sys_name}-{version}")
             }
         )
+    possible_sys_and_versions = set(
+        map(resolve_deprecated_target, possible_sys_and_versions)
+    )
```

Comment:
it's a matter of taste, but again, I like comprehensions more

```suggestion
    possible_sys_and_versions = {
        DEPRECATED_TARGET_MAP.get(target, target)
        for target in possible_sys_and_versions
    }
```

Reply:
Thank you for suggestions. Despite I like higher-order functions more than comprehensions, I'm going to follow your suggestions to keep the code consistent.